CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.6.0 (2020-09-16)
+
+- Added the `theme` property to the `SkyDocsCodeExampleComponent` to allow loading Stackblitz demos in modern theme. [#130](https://github.com/blackbaud/skyux-docs-tools/pull/130)
```

Comment:
You can delete "allow" and change "loading" to "load."

Reply:
Done!